exo-run/src/service-runner.ls

```diff
@@ -76,6 +76,8 @@ class ServiceRunner extends EventEmitter
     dependencies = []
     for dependency-name, dependency-config of @service-config.dependencies or {}
       container-name = "#{@config.app-name}-#{dependency-name}"
+      if dependency-config?.version?
+        dependency-name += ":#{dependency-config.version}"
```

Comment:
Good catch. You might want to remove the trailing `?` from line 79 to make sure you get the actual version then.